plugins/systemd/systemd_units

```diff
@@ -51,6 +68,10 @@ states="active \
 	failed \
 	activating \
 	deactivating"
+
+include="${include:-.*}"
+exclude="${exclude:-^$}"
```

Comment:
The `include` and `exclude` filters work exclusively on unit names (e.g., `apache2.service`, `my-cron.timer`, etc.). Defaulting the exclude to `^$` (empty string) simply makes it so that it _never_ matches a unit file name, and thus will not exclude anything by default. It's not a problem if the user overrides this with anything else.

Reply:
OK - thanks for the clarification. Probably I would have used an empty string to indicate, that the filter should be ignored. Anyway - just a matter of taste.